From: Stefano Zacchiroli <zack@upsilon.cc>
Date: Sat, 3 May 2008 22:39:14 +0200
Subject: [PATCH] bug fix: libinstall-b-c should not depend on native code

Signed-off-by: Stefano Zacchiroli <zack@upsilon.cc>
---
 OCamlMakefile |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/OCamlMakefile b/OCamlMakefile
index b0b9252..b51c452 100644
--- a/OCamlMakefile
+++ b/OCamlMakefile
@@ -1170,7 +1170,7 @@ libinstall:	all
 	$(QUIET)printf "\nInstallation successful.\n"
 
 .PHONY: libinstall-byte-code
-libinstall-byte-code:	all
+libinstall-byte-code: byte-code-library
 	$(QUIET)printf "\nInstalling byte-code library with ocamlfind\n"
 	$(OCAMLFIND) install $(OCAMLFIND_INSTFLAGS) $(RESULT) META \
 	  $(filter-out $(RESULT).$(EXT_LIB) $(RESULT).cmxa, $(LIBINSTALL_FILES))
-- 
