From: Stefano Zacchiroli <zack@upsilon.cc>
Date: Sat, 3 May 2008 22:39:34 +0200
Subject: [PATCH] extra targets to enable compilation of byte code only

Signed-off-by: Stefano Zacchiroli <zack@upsilon.cc>
Signed-off-by: Stephane Glondu <steph@glondu.net>
---
 Makefile     |   10 ++++++++++
 lib/Makefile |    1 +
 2 files changed, 11 insertions(+), 0 deletions(-)

diff --git a/Makefile b/Makefile
index 04a3d60..5a12487 100644
--- a/Makefile
+++ b/Makefile
@@ -17,3 +17,13 @@ uninstall:
 clean:
 	@cd lib && $(MAKE) clean
 	@cd extended && $(MAKE) clean
+
+.PHONY: byte
+byte:
+	@cd lib && $(MAKE) byte
+	@cd lib_test && make
+
+.PHONY:	libinstall-byte-code
+libinstall-byte-code:
+	@cd lib && $(MAKE) $@
+
diff --git a/lib/Makefile b/lib/Makefile
index ab1c8d8..d2af679 100644
--- a/lib/Makefile
+++ b/lib/Makefile
@@ -210,6 +210,7 @@ THREADS = yes
 ANNOTATE = true
 
 all: byte-code-library native-code-library
+byte: byte-code-library
 
 LIBINSTALL_FILES = \
   core.cmi core.cma core.cmxa core.a libcore_stubs.a dllcore_stubs.so
-- 
1.6.5.2

