From: Stefano Zacchiroli <zack@upsilon.cc>
Date: Mon, 9 Nov 2009 13:23:05 +0100
Subject: [PATCH] Fix build on kfreebsd

---
 lib/Makefile |    6 +++++-
 1 files changed, 5 insertions(+), 1 deletions(-)

diff --git a/lib/Makefile b/lib/Makefile
index d2af679..40eabae 100644
--- a/lib/Makefile
+++ b/lib/Makefile
@@ -1,9 +1,13 @@
 OCAMLMAKEFILE = ../OCamlMakefile
 
-ifeq ($(shell uname -s), Linux)
+SYSTEM := $(shell uname -s)
+
+ifeq ($(SYSTEM), Linux)
   LINUX_EXT_STUBS_C = linux_ext_stubs.c
   LINUX_EXT_ML = linux_ext.mli linux_ext.ml
 	CLIBS += rt
+else if ($(SYSTEM), GNU/kFreeBSD)
+  CLIBS = rt
 endif
 
 SOURCES = \
-- 
