Fix time test.

--- a/core/lib_test/time_test.ml
+++ b/core/lib_test/time_test.ml
@@ -56,7 +56,7 @@
       "sexp1"  @? (Time.t_of_sexp (Time.sexp_of_t time1) = time1);
       "sexp2"  @? (Time.t_of_sexp (Time.sexp_of_t time2) = time2);
       "sexp3"  @? (Time.t_of_sexp (Time.sexp_of_t time3) = time3);
-      let date, ofday = Time.to_local_date_ofday time3 in
+      let date, ofday = Time.to_date_ofday time3 (Zone.of_string "nyc") in
       "date"   @? (date = Date.of_string "2005-05-25");
       "ofday"  @? (Ofday.(=.) ofday (Time.Ofday.of_string "12:46:15.232"));
       "ofday1" @? (Time.Ofday.of_string "09:13" = Time.Ofday.of_string "0913");
